README.md

```diff
@@ -47,10 +47,10 @@ Reference the library inside of your `.html` page using:
 
 ```html
 <!-- Microsoft Teams JavaScript API (via CDN) -->
-<script src="https://unpkg.com/@microsoft/teams-js@1.4.1/dist/MicrosoftTeams.min.js" integrity="sha384-wHgBQlRj8iDw76cpan9ViEoOBiIJid4ACaE6vA2gUJPtn15GJnMmUGS+fLxKBWeI" crossorigin="anonymous"></script>
+<script src="https://statics.teams.microsoft.com/sdk/v1.4.3/js/MicrosoftTeams.min.js" crossorigin="anonymous"></script>
```

Comment:
v1.4.3 isn’t on the CDN yet; let’s point to v1.4.2 instead. The v1.4.3 release only ads changes specific to the way that the SDK gets consumed via the npm package.

Reply:
1.4.3 isn't on the CDN, but we've published it to npm? https://www.npmjs.com/package/@microsoft/teams-js

Reply:
Yup, this is the exact issue we were trying to address by using unpkg. Deploying to our CDN is a tedious manual process at the moment. @sachingupta can help get v1.4.3 published but it might take some wheeling and dealing.

Reply:
Ah, ok.

Reply:
I should have tried the URL :)

Reply:
HI @clearab We just published V1.4.3 to teams cdn today. Please use 
<script src="https://statics.teams.microsoft.com/sdk/v1.4.3/js/MicrosoftTeams.min.js" integrity="sha384-3eIlLKLlRvFpO327eFF+Hh/I0QpH4BxuCGYVJ9gemIKuaOau32pBPR2sBGoWd2UO" crossorigin="anonymous"></script>

Reply:
@clearab, will you be able to make this update to your PR so that we can get the change committed?